### DISCUSSION TITLE
How we get background location on iOS Device using React Native

### DISCUSSION THREAD

**Discussion by asadmumtaz92** [community/community#50647] [Mobile]
### Select Topic Area

General

### Body

Hey I hope You are all going well.

I try for many library for getting background location for iOS device.
some library are below but I failed to get location when user close app.
If app us open then its working file.

1. https://www.npmjs.com/package/react-native-geolocation-service
2. https://www.npmjs.com/package/react-native-background-fetch
3. https://www.npmjs.com/package/react-native-get-location

Please anyone suggest me solution or any library that work for both android and iOS platfoem.

Thanks!

**Comment by aconsuegra** [comment]
Hi @asadmumtaz92! We use the "Mobile" category in this discussion community to ask questions, reports bugs or request features in GitHub Mobile. I will close this discusison because it is not related to the native GitHub apps.
